automatic/jenkins/ReadMe.md

```diff
@@ -0,0 +1,29 @@
+# ![Jenkins Logo](https://cdn.jsdelivr.net/gh/pauby/chocopackages@ae92e032839a27a3633db7018593b075df44269f/icons/jenkins.png "Jenkins Logo") [Jenkins](https://chocolatey.org/packages/jenkins)
```

Comment:
To have the URL here in the Readme be updated automatically, change it to use a similar header as the other packages in the repository:

```suggestion
# <img src="https://cdn.jsdelivr.net/gh/pauby/chocopackages@ae92e032839a27a3633db7018593b075df44269f/icons/jenkins.png" width="48" height="48"/> [Jenkins](https://chocolatey.org/packages/jenkins)
```